assets/js/util.js

```diff
@@ -119,6 +119,25 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Comment:
Use method chaining: https://schier.co/blog/2013/11/14/method-chaining-in-javascript.html